internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -223,19 +228,42 @@ Resources:
             Type: Map
             End: true
             ItemsPath: $.GetMappingFile.files
-            ItemSelector:
-              mapping.$: $$.Map.Item.Value
-              destBucket.$: $.DestinationBucket
-              srcBucket.$: $.SourceBucket
             ItemProcessor:
               ProcessorConfig:
                 Mode: INLINE
-              StartAt: CopyFile
+              StartAt: ContentTypeChoice
               States:
+                ContentTypeChoice:
+                  Type: Choice
+                  Choices:
+                    - Or:
+                      - Variable: $.contentType
+                        IsPresent: false
+                      - Variable: $.contentType
+                        StringMatches: ""
+                      Next: CopyFile
+                  Default: CopyFileWithContentType
                 CopyFile:
                   Type: Task
                   End: true
-                  Resource: !GetAtt CopyAssetsFunction.Arn
+                  Resource: arn:aws:states:::aws-sdk:s3:copyObject
+                  Parameters:
+                    CopySource.$: States.Format('{{.AssetMappingFileBucket}}/{}', $.path)
+                    Bucket: !Ref Bucket
+                    Key.$: $.destPath
+                    MetadataDirective: "REPLACE"
```

Comment:
wait if without content type why do we need this?

Reply:
theoretically, if `contentType` was empty, then we should reset the `Content-Type` of the file to the default. example:
1. upload `index.html`, content type in s3 is `text/html`
2. rename `index.html` -> `index.xyz` on local machine without changing the file's content
3. upload `index.xyz`, detected content type is `""`, s3 content type should change to `binary/octet-stream` instead of staying `text/html`